matplotlib-tutorial.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Are you kidding me...?</p>

 _Reply via <a href='https://app.reviewnb.com/sebinsua/test-review-notebooks/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>